web/src/message_list_tooltips.ts

```diff
@@ -132,16 +139,19 @@ export function initialize() {
             }
             const $elem = $(instance.reference);
             const local_id = $elem.attr("data-reaction-id");
-            const message_id = rows.get_message_id(instance.reference);
+            assert(local_id !== undefined);
+            // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+            const message_id = rows.get_message_id(instance.reference as HTMLElement);
```

Comment:
It's better to just do an assert that `instance.reference` is an HTMLElement than do an untyped cast; that'll avoid the need for an `eslint` disable here.